toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/GeoViewScope.kt

```diff
@@ -134,6 +135,10 @@ public sealed class GeoViewScope protected constructor(private val geoView: GeoV
             allowCalloutRecomposition = true
             this.CalloutInternal(location, modifier, offset, rotateOffsetWithGeoView, colorScheme, shapes, content)
         }
+
+        SideEffect {
+            reset()
+        }
```

Comment:
suggest to move this up into the if block above.

```suggestion

        SideEffect {
            reset()
        }
```